test/runtime/Policies.go

```diff
@@ -633,27 +633,25 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 
 	It("Checks CIDR Policy", func() {
 
-		ipv4Host := "192.168.254.254"
 		ipv4OtherHost := "192.168.254.111"
 		ipv4OtherNet := "99.11.0.0/16"
-		ipv6Host := "fdff::ff"
 		httpd2Label := "id.httpd2"
 		httpd1Label := "id.httpd1"
 		app3Label := "id.app3"
 
-		log.Infof("IPV4 Address Host: %s", ipv4Host)
+		log.Infof("IPV4 Address Host: %s", helpers.IPv4Host)
```

Comment:
Why not `log.WithFields`, is not clear less verbose?

Reply:
The purpose of the log message itself is just to log a value, so I don't see the point of using a field.